dataland-frontend/src/components/forms/parts/kpiSelection/EuTaxonomyKPIsModel.ts

```diff
@@ -172,7 +172,7 @@ export const euTaxonomyKpiInfoMappings = {
   sustainabilityReportDate: "The date until which the information presented in the report is valid.",
   sustainabilityReportCurrency: "The 3-letter alpha code that represents the currency used in the report.",
   integratedReport: "Link to Integrated Report.",
-  groupLevelIntegratedReport: "Is the Integrated Report on a Group level?",
+  groupLevelIntegratedReport: "Is the report on a group level?",
```

Comment:
Is changing this one necessary?

Reply:
It does not look so. We should check what is actually needed in this file. I feel, this whole block is completely unused.